SignalMessaging/categories/UIColor+OWS.h

```diff
@@ -8,21 +8,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface UIColor (OWS)
 
-+ (UIColor *)ows_signalBrandBlueColor;
-+ (UIColor *)ows_materialBlueColor;
-+ (UIColor *)ows_destructiveRedColor;
-+ (UIColor *)ows_fadedBlueColor;
-+ (UIColor *)ows_darkBackgroundColor;
-+ (UIColor *)ows_darkGrayColor;
-+ (UIColor *)ows_yellowColor;
-+ (UIColor *)ows_reminderYellowColor;
-+ (UIColor *)ows_reminderDarkYellowColor;
-+ (UIColor *)ows_greenColor;
-+ (UIColor *)ows_redColor;
-+ (UIColor *)ows_blackColor;
-+ (UIColor *)ows_errorMessageBorderColor;
-+ (UIColor *)ows_infoMessageBorderColor;
-+ (UIColor *)ows_inputToolbarBackgroundColor;
+@property (class, readonly, nonatomic) UIColor *ows_systemPrimaryButtonColor;
```

Comment:
class properties just read a little more idiomatically for things like this in Swift.